skyportal/handlers/api/photometry.py

```diff
@@ -592,6 +603,28 @@ def get_group_ids(self):
         group_ids = list(set(group_ids))
         return group_ids
 
+    def get_stream_ids(self):
+        data = self.get_json()
+        stream_ids = data.pop("stream_ids", [])
+        if isinstance(stream_ids, (list, tuple)):
```

Comment:
does `get_json` sometimes return a tuple and sometimes a list? Might want to explain the isinstanace check here.

Reply:
This is the pattern used throughout the rest of the photometry handlers

Reply:
Leaving as is for consistency